import.Rmd

```diff
@@ -12,7 +12,9 @@ library(readr)
 
 ## Overview
 
-You can't apply any of the tools you've applied so far to your own work, unless you can get your own data into R. In this chapter, you'll learn how to import:
+To apply the tools we've discussed in your work, you'll first have to get your own data into R.
+
```

Comment:
The sentence feels like it was a bit of a mouthful. I've trimmed it down a bit, and hopefully haven't lost much of the meaning.